app/controllers/api/authentication_controller.rb

```diff
@@ -0,0 +1,12 @@
+class Api::AuthenticationController < Api::ApiController
+  skip_before_action :authenticate_token!
+
+  def create
+    user = User.find_by(email: params[:user][:email])
+    if user.valid_password? params[:user][:password]
+      render json: { token: JsonWebToken.encode(sub: user.id) }
+    else
+      render json: { errors: ['Invalid email or password'] }
+    end
```

Comment:
- [ ] Good job on making all these API endpoints 👏 👏 However from the project requirements, you are expected to create an API endpoint for **user registration** --> POST: `/register` or `/signup` Therefore, I kindly suggest that you can create it too 😃 👍

Reply:
@BertrandConxy kindly can you check the requirements correctly I don't see anything that say the creation of user.
[Here](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_api_endpoints.md#project-requirements)